### DISCUSSION TITLE
I installed a script on Windows 10 Home and lost access to my PC

### DISCUSSION THREAD

**Discussion by xFELAx** [simeononsecurity/Windows-Optimize-Harden-Debloat#41] [Should've been made as an issue] [answered]
Hi! I installed this script on my Windows 10 Home. I didn't read the requirements and other crucial things ( I'm a stupid student :/ ). After rebooting I can't login into my microsoft account. When I try to login there is a message: "Something went wrong and your PIN isn't available (status: 0xc000015b, substatus:0x0 Restart your device to see if that fixes the problem". I searched for the fix online but nothing helped. When I try to do something in a safe mode I realized that there is only available a new X_Admin account but I don't know the password for this account. Could you help me to regain access to my PC? Sorry for a problem. :/

**Comment by simeononsecurity** [comment, marked as answer]
Microsoft account signins are not supported by this script or by DoD STIGs, which are documented in the [readme](https://github.com/simeononsecurity/Windows-Optimize-Harden-Debloat) of this repo. Additionally, you are to test this script on a VM or clean install before ever running in on any personal system.
The X_Admin account is to satisfy the hardening recommendations and requirements to lock down the local administrator account. This is not your account, it is a built in default account of windows installs. Unless you've previously configured it, it does not have a password and the account is disabled via GPO.

Unfortunately here, the best option you have is to reinstall windows.
